### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Yhugzee**

### CODE REVIEW

File: frontend/src/components/NavBar.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export default function NavBar() {
+  return (
+    <div className="NavBar">
```
Comment (rocambille) [inline frontend/src/components/NavBar.jsx:3]: if it's a navbar, why not a nav ?

File: frontend/src/components/NavBar.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      <div className="NavBar_links">
+        <ul>
```
Comment (rocambille) [inline frontend/src/components/NavBar.jsx:5]: try to put the class on the ul (and then you could remove the div)

File: frontend/src/components/NavBar.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+          <li>
+            <a href="/">Home</a>
+            <a href="/about">About</a>
+            <a href="/contact">Contact</a>
```
Comment (rocambille) [inline frontend/src/components/NavBar.jsx:9]: each link should be in a li

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/NavBar.jsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/button-has-type */
+// Pour Romain ou Justine : Si je n'ai pas la ligne pour désactiver le linter, j'ai une erreur eslint qui me dit que le type du bouton doit être défini. Je ne sais pas pourquoi, mais je n'ai pas eu ce problème avec le code de base. Je n'ai pas trouvé de solution pour le moment.
+import React, { useState, useEffect } from "react";
+import "./NavBar.css";
+
+export default function NavBar() {
+  const [toggleMenu, setToggleMenu] = useState(false);
+  const [largeur, setLargeur] = useState(window.innerWidth);
```
Comment (rocambille) [inline frontend/src/components/NavBar.jsx:8]: in english

File: frontend/src/components/NavBar.jsx [hunk 5]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/button-has-type */
+// Pour Romain ou Justine : Si je n'ai pas la ligne pour désactiver le linter, j'ai une erreur eslint qui me dit que le type du bouton doit être défini. Je ne sais pas pourquoi, mais je n'ai pas eu ce problème avec le code de base. Je n'ai pas trouvé de solution pour le moment.
```
Comment (rocambille) [inline frontend/src/components/NavBar.jsx:2]: so put a type to your button (and remove this eslint line)

https://html.com/attributes/button-type/

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/NavBar.jsx (hunk 4 above)
Comment (Yhugzee) [reply frontend/src/components/NavBar.jsx:8]: ✅

**Pull request merged**